tests/integration/test_iam.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from databricks.sdk.core import DatabricksError
+
+
+def test_filtering_groups(w, random):
+    all = w.groups.list(filter=f'displayName eq any-{random(12)}')
+    found = len(list(all))
+    assert found == 0
+
+
+def test_scim_error_unmarshall(w, random):
+    with pytest.raises(DatabricksError) as exc_info:
+        w.groups.list(filter=random(12))
+    assert 'Given filter operator is not supported' in str(exc_info.value)
+
+
+@pytest.mark.parametrize("path,call", [
+    ("/api/2.0/preview/scim/v2/Users", lambda w: w.users.list(count=1)),
+    ("/api/2.0/preview/scim/v2/Groups", lambda w: w.groups.list(count=1)),
+    ("/api/2.0/preview/scim/v2/ServicePrincipals", lambda w: w.service_principals.list(count=1)),
```

Comment:
```suggestion
    ("/api/2.0/preview/scim/v2/Users", lambda w: w.users.list(count=10)),
    ("/api/2.0/preview/scim/v2/Groups", lambda w: w.groups.list(count=10)),
    ("/api/2.0/preview/scim/v2/ServicePrincipals", lambda w: w.service_principals.list(count=10)),
```

i think count=1 would make integration tests too slow...

Reply:
I tried this, but I wanted to make sure that we actually exercised the pagination flows. This test took around 15 seconds for all test cases combined.